tests/components/rust/pinger-config-component/src/lib.rs

```diff
@@ -44,8 +44,8 @@ impl exports::wasi::http::incoming_handler::Guest for Actor {
         let pong_secret = pingpong::ping_secret();
 
         let res = json!({
-            "single_val": config::runtime::get(&config_key).expect("failed to get config value"),
-            "multi_val": config::runtime::get_all().expect("failed to get config value").into_iter().collect::<HashMap<String, String>>(),
+            "single_val": config::store::get(&config_key).expect("failed to get config value"),
+            "multi_val": config::store::get_all().expect("failed to get config value").into_iter().collect::<HashMap<String, String>>(),
```

Comment:
Thoughts on retaining this one as `config::runtime` in order to confirm that backwards compat is working as expected (and we can file a deprecation issue?

Reply:
Added a test case in https://github.com/wasmCloud/wasmCloud/pull/3312/commits/51ea896fe0227d77765dcdb742b9a772c6ecdbfd, since this component does not have any other imports other that `wasmcloud_component` crate, I'd rather keep it that way